test/units/testsuite-29.sh

```diff
@@ -5,6 +5,8 @@
 set -eux
 set -o pipefail
 
+export SYSTEMD_DISSECT_VERITY_TIMEOUT_SEC=30
```

Comment:
this will only affect the local commands, no? but it's needed for PID1 too?

Reply:
Ah,,, indeed...

Reply:
and also for systemd-portabled.service